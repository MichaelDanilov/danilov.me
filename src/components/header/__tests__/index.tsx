import React from 'react';
import renderer from 'react-test-renderer';
import Header from '..';

describe('Header', () => {
  it('english version renders correctly', () => {
    const tree = renderer.create(<Header lang="en" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('russian version renders correctly', () => {
    const tree = renderer.create(<Header lang="ru" />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('add custom about', () => {
    const tree = renderer
      .create(<Header lang="en" about="Lorem ipsum" />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
