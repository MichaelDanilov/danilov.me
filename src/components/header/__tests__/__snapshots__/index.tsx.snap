// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header add custom about 1`] = `
Array [
  <h1
    className="sc-bdVaJa gyVQum"
  >
    Michael Danilov
  </h1>,
  <p
    className="sc-bwzfXH jfgEZP"
    dangerouslySetInnerHTML={
      Object {
        "__html": "Lorem ipsum",
      }
    }
  />,
]
`;

exports[`Header english version renders correctly 1`] = `
<h1
  className="sc-bdVaJa gyVQum"
>
  Michael Danilov
</h1>
`;

exports[`Header russian version renders correctly 1`] = `
<h1
  className="sc-bdVaJa gyVQum"
>
  Михаил Данилов
</h1>
`;
